tests/test_arrow_dataset.py

```diff
@@ -1632,6 +1632,25 @@ def __call__(self, example):
                 dset.map(ex_cnt)
                 self.assertEqual(ex_cnt.cnt, len(dset))
 
+    def test_map_crash_subprocess(self, in_memory):
```

Comment:
maybe skip the test on windows ? it seems to raise a PermissionError
```suggestion
    @require_not_windows
    def test_map_crash_subprocess(self, in_memory):
```